src/components/TasksTable/index.jsx

```diff
@@ -245,15 +246,30 @@ export default class TasksTable extends Component {
     this.setQuery({});
   };
 
-  handleDrawerOpen = ({ target: { name } }) => {
+  handleDrawerOpen = ({ target: { name } }, client) => {
```

Comment:
Why are we passing `client` here?

Reply:
because we need it to do client.query?

Reply:
Can you do `this.props.client.query()` inside `src/views/Project/index.jsx` instead?